plugins/module_utils/oneview.py

```diff
@@ -1487,6 +1488,20 @@ def _replace_volume_attachment_names_by_uri(self, data):
                                                   self.oneview_client.storage_pools,
                                                   replace_name_with='')
 
+    def _replace_volume_attachment_connection_name_by_uri(self, data):
+        volume_attachments = (data.get('sanStorage') or {}).get(
+            'volumeAttachments') or []
+        if len(volume_attachments) > 0:
+            for volume in volume_attachments:
+                storage_paths = volume.get('storagePaths') or []
+                if len(storage_paths) > 0:
+                    for storage_path in storage_paths:
+                        if storage_path.get('networkName'):
+                            network_name = storage_path['networkName']
+                            del storage_path['networkName']
+                            if network_name is not None:
+                                storage_path['networkUri'] = self._get_network_by_name(network_name)['uri']
```

Comment:
what happen if we did not find resource by name(provided by user)? will we show some error or it goes to OV and OV throws error

Reply:
It will go to OV and OV throws the error